### PR TITLE
Ui updates

### PR DESCRIPTION
**Pull request by JourneyToTheWes**
**Pull Request Updates (UI Updates)**
- [x] Updated dwell time on key selection and made time consistent on all other buttons requiring longer dwell. 
- [x] Also updated major/minor button selection size responsive on different screen sizes.
- [x] Text for functional inputs on the piano page are responsive now also.
- [x] Made #/b replaced with unicode escape sequences for music sharp and flat characters

Resolving issues: #11 and #32

### CODE REVIEW

File: GazePianoPrototype/KeySelector.xaml [hunk 1]
```diff
@@ -27,31 +28,31 @@
...
+        <Button Grid.Row="3" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="C&#x0266F;" Tag="C#" Click="Button_Click"/>
+        <Button Grid.Row="3" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="C&#x0266D;" Tag="Cb" Click="Button_Click"/>
 
-        <Button Grid.Row="4" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="D" Click="Button_Click"/>
-        <Button Grid.Row="4" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="D#" Click="Button_Click"/>
-        <Button Grid.Row="4" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="Db" Click="Button_Click"/>
+        <Button Grid.Row="4" Grid.Column="0" Style="{StaticResource KeySelectionButton}" Content="D" Tag="D" Click="Button_Click"/>
+        <Button Grid.Row="4" Grid.Column="1" Style="{StaticResource KeySelectionButton}" Content="D&#x0266F;" Tag="D#" Click="Button_Click"/>
+        <Button Grid.Row="4" Grid.Column="2" Style="{StaticResource KeySelectionButton}" Content="Db" Tag="Db" Click="Button_Click"/>
```
Comment (parkerreno) [inline GazePianoPrototype/KeySelector.xaml]: The b in Db should be swapped to a flat symbol as well

**Review by parkerreno — CHANGES_REQUESTED** [review]
One minor change requested, then good to merge

File: GazePianoPrototype/KeySelector.xaml (hunk 1 above)
Comment (JourneyToTheWes) [reply GazePianoPrototype/KeySelector.xaml]: Thanks for catching that. Okay it's resolved now.

Comment (parkerreno) [reply GazePianoPrototype/KeySelector.xaml]: Awesome. Just waiting for the build to complete and then I'll complete the pr.

**Review by parkerreno — APPROVED** [review]

**Pull request merged**